test/e2e/specs/onboarding/setup__start-writing.ts

```diff
@@ -0,0 +1,154 @@
+/**
+ * @group calypso-release
+ */
+import {
+	DataHelper,
+	UserSignupPage,
+	BrowserManager,
+	NewUserResponse,
+	RestAPIClient,
+	DomainSearchComponent,
+	EditorPage,
+	CartCheckoutPage,
+} from '@automattic/calypso-e2e';
+import { Page, Browser } from 'playwright';
+import { apiCloseAccount } from '../shared';
+
+declare const browser: Browser;
+
+describe( DataHelper.createSuiteTitle( 'Start Writing Tailored Onboarding' ), () => {
+	const testUser = DataHelper.getNewTestUser( {
+		usernamePrefix: 'start_writing',
+	} );
+	let page: Page;
+	let domainSearchComponent: DomainSearchComponent;
+	let newUserDetails: NewUserResponse;
+	let cartCheckoutPage: CartCheckoutPage;
+
+	beforeAll( async () => {
+		page = await browser.newPage();
+		await BrowserManager.setStoreCookie( page, { currency: 'EUR' } );
+	} );
+
+	describe( 'Signup via /setup/start-writing', function () {
+		let editorPage: EditorPage;
+		const postTitle = 'my first post title';
+
+		it( 'Navigate to /setup/start-writing', async function () {
+			await page.goto( DataHelper.getCalypsoURL( '/setup/start-writing' ) );
+		} );
+
+		it( 'Enter account details', async function () {
+			await page.waitForURL( /.*start-writing.*/ );
+			const userSignupPage = new UserSignupPage( page );
+			newUserDetails = await userSignupPage.signup(
+				testUser.email,
+				testUser.username,
+				testUser.password
+			);
+		} );
+
+		it( "Start user's first post", async function () {
+			await page.waitForURL( /.*post-new.php\?start-writing=true.*/ );
+			editorPage = new EditorPage( page );
+		} );
+
+		it( 'Enter blog title', async function () {
+			await editorPage.enterTitle( postTitle );
+		} );
+
+		it( 'Publish post and open Launchpad', async function () {
+			await editorPage.publish();
+			await page.waitForURL( /.*start-writing\/launchpad.*/ );
+			await page.locator( ':text("Your blog\'s almost ready!")' ).waitFor();
+		} );
+
+		it( 'Add blog name and description', async function () {
+			await page.getByRole( 'button', { name: 'Name your blog' } ).click();
+			await page.getByPlaceholder( 'A catchy name to make your blog memorable' );
+			await page.fill(
+				'input[name="setup-form-input-name"]',
+				`Start writing site ${ testUser.username }`
+			);
+			await page.fill(
+				'textarea[name="setup-form-input-description"]',
+				`The place of ${ testUser.username }`
+			);
+			await page.click( 'button.setup-form__submit' );
+		} );
+
+		it( 'Navigate choose a domain', async function () {
+			await page.waitForURL( /.*start-writing\/launchpad.*/ );
+			await page.getByRole( 'button', { name: 'Choose a domain' } ).click();
+		} );
+
+		it( 'Search for a domain', async function () {
+			domainSearchComponent = new DomainSearchComponent( page );
+			await domainSearchComponent.search( testUser.username );
+		} );
+
+		it( 'Skip selecting a domain', async function () {
+			await Promise.all( [ page.waitForNavigation(), page.click( 'text=Decide later' ) ] );
```

Comment:
**issue:blocking** - `waitForNavigation` is deprecated and not suggested for use by upstream. `waitForURL` is a suitable alternative that they suggest.

Reply:
Nice, [fixed here](https://github.com/Automattic/wp-calypso/pull/77457/commits/3e3a261f10165713674c4aef7d5ce34961542e78).